roles/reset/tasks/main.yml

```diff
@@ -80,6 +81,7 @@
     - crictl.stat.exists
     - container_manager in ["crio", "containerd"]
     - deploy_container_engine
+    - ansible_facts.services['containerd.service'] is defined
```

Comment:
Won't this always be false on cri-o setups ?

Reply:
thanks @floryut  , updated PR, forgot about cri-o, please take a look one more time